modules/microvm/modules.nix

```diff
@@ -185,23 +185,25 @@ in
       netvm.extraModules =
         optionals cfg.netvm.enable [
           serviceModules.logging
+          serviceModules.givc
```

Comment:
im surprised this build on aarch64, did not know that our givc was ready for these targets. so lets test it and see if it blows up :)

Reply:
Thanks @emrahbillur this is great!